lib/WPStrava/SOM.php

```diff
@@ -49,14 +59,18 @@ public function get_swimpace_label() {
 	/**
 	 * Change meters per second to Minutes Per 100 Meters. Same for English/metric.
 	 *
-	 * @param float $mps Meters per second.
-	 * @return float Minutes Per 100 Meters.
+	 * @param float|string $mps Meters per second.
+	 * @return string Minutes Per 100 Meters.
 	 */
 	public function swimpace( $mps ) {
```

Comment:
@sebastianerb I changed this to be more like the regular pace functions (min:sec per distance) because this was displaying "1.5" as a 1 minute 30 second per 100m pace. I'm not a swimmer, so I want to confirm with you that it's OK. Will merge now, but can revert if needed.

Reply:
@jrfoell yes absolutely fine with this!